### PR TITLE
change def of weighted transform

### PR DESCRIPTION
**Pull request by cxhernandez**
re: https://github.com/msmbuilder/msmbuilder/issues/605

### CODE REVIEW

**Comment by cxhernandez** [comment]
There's some random error in `msmbuilder.tests.test_ghmm.test_3_state` for appveyor

**Comment by cxhernandez** [comment]
Also as a side note: I'm not sure this is as tiny a change as @rmcgibbo made it out to be. The convergence of my implied timescales improved greatly for my system after using this, but the slowest characteristic timescale was reduced by about 5-fold, which was way more than the 95% confidence interval around that value from bootstrapping. Currently running a bunch of osprey jobs to make sure this wasn't a fluke, but something to keep in mind.

**Comment by mpharrigan** [comment]
How about we add a new option (`kinetic_distance` ?) and add a warning that `weighted_transform` is deprecated and people should use `kinetic_distance`. And add a test and update the changelog

**Comment by cxhernandez** [comment]
more appveyor weirdness

File: msmbuilder/decomposition/tica.py [hunk 1]
```diff
@@ -42,8 +42,10 @@ class tICA(BaseEstimator, TransformerMixin):
         specified (the default) it is estimated using an analytic formula
         (the Rao-Blackwellized Ledoit-Wolf estimator) introduced in [5].
     weighted_transform : bool, default=False
-        If True, weight the projections by the implied timescales, giving
-        a quantity that has units [Time].
+        Deprecated. Please use kinetic_mapping.
+    kinetic_mapping : bool, default=False
```
Comment (mpharrigan) [inline msmbuilder/decomposition/tica.py]: Add a ref to frank's paper?

File: msmbuilder/decomposition/tica.py [hunk 2]
```diff
@@ -321,7 +325,12 @@ def transform(self, sequences):
             X_transformed = np.dot(X, self.components_.T)
 
             if self.weighted_transform:
+                warnings.warn("weighted_transform is deprecated. "
+                              "Please use kinetic_mapping",
+                              DeprecationWarning)
                 X_transformed *= self.timescales_
+            elif self.kinetic_mapping:
```
Comment (mpharrigan) [inline msmbuilder/decomposition/tica.py]: If both are set to `True`, can you raise an error?

File: msmbuilder/decomposition/tica.py [hunk 3]
```diff
@@ -448,6 +457,7 @@ def summarize(self):
 shrinkage           : {shrinkage}
 lag_time            : {lag_time}
 weighted_transform  : {weighted_transform}
+kinetic_mapping : {kinetic_mapping}
```
Comment (mpharrigan) [inline msmbuilder/decomposition/tica.py]: Can you space pad this so it's aligned like the other fields?

**Comment by brookehus** [comment]
Don't mind me, just listening.

**Comment by mpharrigan** [comment]
lgtm

**Pull request merged**